package.json

```diff
@@ -33,7 +33,7 @@
     "test": "npm run build && npm run test:fast && npm run catch-uncommitted",
     "test:local-visual": "./scripts/test-screenshots.sh",
     "test:visual": "npm run ci:screenshot -- -o .reg/current && reg-cli .reg/current __screenshots__ .reg/diff -R .reg/report.html",
-    "jest": "jest __tests__ --maxWorkers=4",
+    "jest": "npm run build && jest __tests__ --maxWorkers=4",
```

Comment:
I don't love this - this means that the `test:fast` command now requires the whole library to be built. If anything we should probably add a distinct separate command for updating screenshots. Additionally, the `test` command already incorporates the `build` step

Reply:
@LucianBuzzo I don't think there are cases when you would want to run tests on a stale build, but I am fine either way. I will add another `update-snapshots` command so we always update snapshots based on a fresh build.